lib/models/garage.dart

```diff
@@ -12,31 +11,38 @@ class Garage {
   List<List<double>> _ticketProbabilities;
 
   Garage({
+    @required this.id,
     @required this.name,
     @required this.location,
     this.enforcementStartTime = const TimeOfDay(hour: 8, minute: 0),
     this.enforcementEndTime = const TimeOfDay(hour: 18, minute: 0),
     this.enforcedOnWeekends = false,
   }) {
-    _initRandomProbabilities();
+    _ticketProbabilities = List.generate(7, (_) => List.generate(24 * 4, (_) => 0.0));
   }
 
-  // TODO: Get enforcement start times and end times from JSON once it is made available
-  // TODO: Get ticket probabilities from JSON once they are made available
   Garage.fromJson(Map<String, dynamic> json) :
+        id = json['pk'],
         name = json['name'],
         location = LatLng(json['latitude'], json['longitude']),
-        enforcementStartTime = TimeOfDay(hour: 8, minute: 0),
-        enforcementEndTime = TimeOfDay(hour: 18, minute: 0),
-        enforcedOnWeekends = false {
-    _initRandomProbabilities();
-  }
-
-  // TODO: Remove once real probabilities are available
-  _initRandomProbabilities() {
-    Random random = Random();
-
-    _ticketProbabilities = List.generate(7, (_) => List.generate(24 * 4, (_) => random.nextDouble()));
+        enforcementStartTime = TimeOfDay(
+          hour: int.parse(json['start_enforce_time'].split(':')[0]),
+          minute: int.parse(json['start_enforce_time'].split(':')[1])
+        ),
+        enforcementEndTime = TimeOfDay(
+          hour: int.parse(json['end_enforce_time'].split(':')[0]),
+          minute: int.parse(json['end_enforce_time'].split(':')[1])
+        ),
+        enforcedOnWeekends = json['enforced_on_weekends'] {
+    _ticketProbabilities = List.generate(
```

Comment:
The format here is pretty hard to read with the braces. Not a big deal and I'm not sure if it can even be changed, but I thought I'd point it out.

Indentation (4 spaces vs. 2 spaces)? 
  - We should try to keep the indentation consistent throughout unless there is some dart/flutter formatting rule I'm not aware of.